packages/grid/x-data-grid-generator/src/services/tree-data-generator.ts

```diff
@@ -11,13 +11,13 @@ export interface AddPathToDemoDataOptions {
 
   /**
    * The depth of the tree
-   * @default: 1
+   * @default 1
    */
   maxDepth?: number;
 
```

Comment:
I wouldn't mind we fix the blank line convention on these files at the same time 😁

```suggestion
```